apps/codelab/src/app/codelabs/angular/angular-cli/angular-cli.component.html

```diff
@@ -1,16 +1,12 @@
 <slide-deck slideShortcuts slidesRouting slides-tracking>
-  <div class="btn-bar">
-    <feedback-widget></feedback-widget>
-    <codelab-menu-shortcut></codelab-menu-shortcut>
-  </div>
   <codelab-progress-bar></codelab-progress-bar>
   <slide-arrows></slide-arrows>
 
   <!--  INTRO -->
   <div *slide id="intro" no-padding>
     <codelab-title-slide
       i18n-title
-      title="angular-cli"
+      title="Angular-cli"
```

Comment:
What's this?

Reply:
Title with a small letter does not look good)
Yeah, it shouldn't be here) I will revert later